### DISCUSSION TITLE
Threejs render inside a render note

### DISCUSSION THREAD

**Discussion by hugosoli** [TriliumNext/Trilium#3670] [Q&A]
Hi!
This may be simple question for some of you guys. There is this library called Threejs which allows to genererate 3d graphics. I want to create Nodes inside trilium that could render such canvas. 
If i create an HTML render node I can insert basic js script but it fails if I tried to use a module.

<script type="module">

  import * as THREE from 'three'; ///parsing errror, import and export should use in sourceType=Module

  const scene = new THREE.Scene();

</script>

If I use a js Child I don know how to specify it should be a module.

Any idea?
cheers

**Comment by zadam** [comment]
TBH I don't know how things are working with modules. Trilium internally uses CommonJS for dependencies and this was the tested approach. Another possibility is to make sure the library exports globally to the `window` object.
